cmake/clang_tidy.cmake

```diff
@@ -5,14 +5,14 @@ if (ENABLE_CLANG_TIDY)
 
     find_program (CLANG_TIDY_CACHE_PATH NAMES "clang-tidy-cache")
     if (CLANG_TIDY_CACHE_PATH)
-        find_program (_CLANG_TIDY_PATH NAMES "clang-tidy-18" "clang-tidy-17" "clang-tidy-16" "clang-tidy")
+        find_program (_CLANG_TIDY_PATH NAMES "clang-tidy-19" "clang-tidy-18" "clang-tidy-17" "clang-tidy-16" "clang-tidy")
```

Comment:
We usually remain compatible with the past three Clang (-tidy) versions. Therefore, remove clang-tidy 17?

Reply:
16*, :ok_hand: